### ISSUE TITLE
Installing in other Linux Distro

### ISSUE THREAD

**Issue by cydrickn** [codesardine/JADESKTOP#119]
Anyone can help on how to install this in other distro like ubuntu

**Comment by codesardine** [comment]
You run it as you would with any python application, however to give it the desktop env experience then you need to add other parts as in building your own distro from the ground up, like a window manager and a dock, I don't work on this for a long time.

**Comment by ghost** [comment]
Hi all! A doubt I have in this open thread would be whether it is possible to use node-os with jde. Is it possible to use node-os with jde?(yes or no) - If yes, how?

**Comment by codesardine** [comment]
> Hi all! A doubt I have in this open thread would be whether it is possible to use node-os with jde. Is it possible to use node-os with jde?(yes or no) - If yes, how?

I don't think so node-os works with node based software and npm as a package manager while this is made in python.
